lib/take_away.rb

```diff
@@ -0,0 +1,56 @@
+class TakeAway
+ 
+
+  attr_writer :menu, :meal_order
+ 
+   def initialize
+     @menu = { "pizza" => 9.99,
+               "cheeseburger" => 4.99,
+               "steak" => 14.99,
+               "salad" => 6.99
+             }
+
+     @meal_order = []
+   end
+
+   def show_menu
+   	@menu.sort{|a,b| a[1]<=>b[1]}.each { |elem| puts " #{elem[0].capitalize}: #{elem[1]}"}
```

Comment:
Metrics/LineLength: Line is too long. [90/80]<br>Rails/Output: Do not write to stdout. Use Rails' logger if you want to log.